server/src/main/java/org/elasticsearch/action/support/broadcast/BroadCastXContentResponse.java

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+package org.elasticsearch.action.support.broadcast;
+
+import org.elasticsearch.action.support.DefaultShardOperationFailedException;
+import org.elasticsearch.common.io.stream.StreamInput;
+import org.elasticsearch.rest.action.RestActions;
+import org.elasticsearch.xcontent.ToXContentObject;
+import org.elasticsearch.xcontent.XContentBuilder;
+
+import java.io.IOException;
+import java.util.List;
+
+public class BroadCastXContentResponse extends BroadcastResponse implements ToXContentObject {
```

Comment:
Good point. I went with `BaseBroadcastResponse` though since we do use the parent class explicitly in a couple spots at the moment and abstract felt wrong. Makes for less noise overall, though the diff looks less nice now for the class itself.